### PR TITLE
Package nsq.0.5.2

### PR DESCRIPTION
**Pull request by ryanslade**
### `nsq.0.5.2`
A client library for the NSQ messaging platform

---
* Homepage: https://github.com/ryanslade/nsq-ocaml
* Source repo: git+https://github.com/ryanslade/nsq-ocaml.git
* Bug tracker: https://github.com/ryanslade/nsq-ocaml/issues

---
:camel: Pull-request generated by opam-publish v2.0.2

### CODE REVIEW

File: packages/nsq/nsq.0.5.2/opam [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+maintainer: "Ryan Slade <ryanslade@gmail.com>"
+authors: "Ryan Slade <ryanslade@gmail.com>"
+homepage: "https://github.com/ryanslade/nsq-ocaml"
+bug-reports: "https://github.com/ryanslade/nsq-ocaml/issues"
+dev-repo: "git+https://github.com/ryanslade/nsq-ocaml.git"
+build: [["dune" "build" "-p" name "-j" jobs]]
+synopsis: "A client library for the NSQ messaging platform"
+depends: [
+  "ocaml" {>= "4.04.0"}
+  "dune" {build}
```
Comment (mseri) [inline packages/nsq/nsq.0.5.2/opam:11]:
```suggestion
  "dune" {>= "2.0"}
```
According to the dune-project file, you require version 2.0 at least.
The `build` tag cannot be used with dune due to forward-dependencies issues.

Comment (mseri) [reply packages/nsq/nsq.0.5.2/opam]: Can you please upstream this change?

Comment (ryanslade) [reply packages/nsq/nsq.0.5.2/opam]: Will do, thanks

**Comment by kit-ty-kate** [comment]
Thanks

**Pull request merged**